upbound_uxp/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f1427ca4-4b88-484a-a7c5-9cc66542b0d9",
+  "app_id": "upbound-uxp",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "Upbound UXP",
+    "media": [],
```

Comment:
Images are required, please add at least one image that follows our [media guidelines](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel).

Reply:
Added reference to image.